.github/workflows/ci.yml

```diff
@@ -179,18 +179,22 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v4
         with:
-          ref: documentation
+          ref: master
 
       - name: Setup
         uses: ./.github/actions/setup
 
       - name: Create Updated Documentation PR
         run: |
-          git checkout -b "documentation-${{ github.sha }}"
-          git merge origin/master --no-edit
-          yarn generate-docs
           git config user.name github-actions
           git config user.email github-actions@github.com
+          git fetch origin
```

Comment:
I just made it super verbose. I also manually synced the `documentation` branch with master just to avoid other conflicts.